src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Arm extends SubsystemBase {
+
+    private final ArmIO arm;
+
+    private final ArmIOInputsAutoLogged armInputs = new ArmIOInputsAutoLogged();
+
+    private boolean upDown;
+
+    public Arm(ArmIO arm) {
+        this.arm = arm;
+    }
+
+    public void armControl(boolean upDown) {
```

Comment:
Controlling this with a boolean doesn't make sense to me - what if we want it to output zero for example?

It would be better to use an enum and set a power based on the state of this subsystem

Reply:
done